raiden/network/transport/matrix/transport.py

```diff
@@ -517,24 +518,30 @@ def whitelist(self, address: Address) -> None:
         # the node that has to create the room.
         self._maybe_create_room_for_address(address)
 
+    def get_user_ids_for_address(self, address: Address) -> Tuple[Set[str], List[User]]:
+        address_hex = to_normalized_address(address)
+        candidates = self._client.search_user_directory(address_hex)
```

Comment:
We should warm the cache for the addresses of `force_check_address_reachability`, it would be better to move the cache warm call from `start_health_check` here.

Reply:
done